charts/consul/values.yaml

```diff
@@ -2201,6 +2201,10 @@ connectInject:
         maxInstances: 1
         minInstances: 1
 
+      # The name of the Openshift SecurityContextConstraints resource to use for Gateways
+      # @type: string
+      openshiftSCCName: "restricted-v2"
```

Comment:
I think it makes sense to include here that this value is only used if `global.openshift.enabled = true`